lib/presentation.js

```diff
@@ -43,7 +43,7 @@ class Presentation {
     }
 
     addSlide(layoutName) {
-        return this.powerPointFactory.addSlide(layoutName || 'slideLayout1');
+        return this.powerPointFactory.addSlide(this, layoutName || 'slideLayout7'); // slide layout 7 is the blank slide layout
```

Comment:
Is this brittle? Maybe we should explicitly cast this to a variable like "blankSlide" so that if it changes at some point we don't need to update all the references.

Reply:
Yeah, I don't like it. Idk why Microsoft couldn't use, gee idk... layout 1 as the blank one! I think we should eventually remove this entirely and put our own blank layout in the XML. That's why I haven't done anything with it yet.